app/test_main.py

```diff
@@ -1,3 +1,27 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize("cat_age,dog_age,expected", [
+        pytest.param(0, 0, [0, 0],
+                     id="Should return zeros, when initial data is zero"),
+        pytest.param(14, 14, [0, 0],
+                     id="Should return zeros, when ages are less than 15"),
+        pytest.param(15, 23, [1, 1],
+                     id="Should return 1 human year, "
+                        "if age of the animal is 15 to 23 inclusive"),
+        pytest.param(24, 24, [2, 2],
+                     id="Should return 2 human years after age of 24"),
```

Comment:
The expected result `[2, 2]` for ages `24, 24` is correct according to the task description, as both cat and dog ages are exactly 24.